R/param-labels.R

```diff
@@ -5,8 +5,8 @@
 #' However it will be extended to parse labels from the model YAML file as well.
 #' The syntax for the labeling is described in "Details" below.
 #'
-#' This function will *not* return indices for the parameters, though they can be added with \code{param_labels() \%>\% apply_indices()}.
-#' See the \code{\link{apply_indices}} documentation for more details.
+#' This function will *not* return indices for the parameters, though they can be added with `param_labels() \%>\% apply_indices()`.
```

Comment:
This renders oddly (the escape character `\` is shown in the result). Maybe consider

```
...though they can be adding by calling `apply_indices()` on the output.
```

Reply:
this was leftover from changing to markdown parsing. Fixed it.